tests/integration_tests/saas/test_stripe_task.py

```diff
@@ -1071,9 +1071,8 @@ def test_stripe_erasure_request_task(
         ],
     )
 
-    # run erasure with MASKING_STRICT to execute the update actions
-
-    config.execution.MASKING_STRICT = True
+    # Run erasure with masking_strict = False so both update and delete actions can be used
+    config.execution.MASKING_STRICT = False
 
```

Comment:
Stripe tests were incorrectly being run with `config.execution.MASKING_STRICT = True` which is invalid for Stripe, because there are both updates and deletes defined in the config.  Nodes with `delete`-only configs were failing silently and then Stripe tests were re-run with `config.execution.MASKING_STRICT = False` below to get delete-specific behavior.

Because a collection no longer fails silently, we were seeing failures here. To address, we can just run a single erasure request with `config.execution.MASKING_STRICT = False` so stripe can use the update if defined, otherwise it uses the delete.  The counts below have been updated to reflect this.